query/reverse.js

```diff
@@ -22,15 +22,9 @@ function generateQuery( clean ){
 
   var vs = new peliasQuery.Vars( defaults );
 
-  // set defaults
-  vs.set({
-    'size': 1,
-    'boundary:circle:radius': '500km'
-  });
```

Comment:
removed this because it should be part of the defaults. now that defaults work properly this should just work.